Tribler/Core/Modules/restapi/market/asks_bids_endpoint.py

```diff
@@ -101,8 +102,14 @@ def render_PUT(self, request):
             request.setResponseCode(http.BAD_REQUEST)
             return json.dumps({"error": "price_type or quantity_type parameter missing"})
 
-        self.get_market_community().create_ask(*BaseAsksBidsEndpoint.create_ask_bid_from_params(parameters))
-        return json.dumps({"created": True})
+        def on_ask_created(_):
+            request.write(json.dumps({"created": True}))
+            request.finish()
```

Comment:
If `request.write` failed, the request will already be finished. Please put this in an `if not request.finished:` statement.